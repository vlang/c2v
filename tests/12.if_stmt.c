int static inline quot( int x, int y ) {
	if( x == 0 || y == 0 )
		return	0;
	if( x > y )
		return	x%y ? 0 : x/y;
    else if (x == y)
        return	x%y ? 0 : x/y;
	else	
		return	y%x ? 0 : y/x;
}
int main() {
	quot(2,2);
}