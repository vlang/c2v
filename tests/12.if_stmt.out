[translated]
module main

fn quot(x int, y int) int {
	if x == 0 || y == 0 {
		return 0
	}
	if x > y {
		return if x % y { 0 } else { x / y }
	} else if x == y {
		return if x % y { 0 } else { x / y }
	} else {
		return if y % x { 0 } else { y / x }
	}
}

fn main() {
	quot(2, 2)
}
